connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/processors/it/ITConstants.java

```diff
@@ -0,0 +1,5 @@
+package com.redhat.cloud.notifications.connector.email.processors.it;
+
+public class ITConstants {
+    public static final String ORG_ADMIN_PERMISSION = "admin:org:all";
```

Comment:
Could you please move that constant to the `ITUserRequest` class since it's not used anywhere else?

Reply:
I can't, because we also use it in the `ITResponseProcessor`